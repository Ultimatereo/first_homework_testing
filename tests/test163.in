3
3.23e-07 2.4e-07 2.71e-07 3e-07
2.8e-07 1.66e-07 2.1e-08 3.67e-07
9.9e-08 4.53e-07 2.48e-07 3e-08
