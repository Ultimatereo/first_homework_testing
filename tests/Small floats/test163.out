1.31405
0.0590763
-0.511504
